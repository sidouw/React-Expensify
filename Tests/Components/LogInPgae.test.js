import React from 'react'
import {shallow} from 'enzyme'
import login from '../../src/Components/LogInPage.jsx'


test('shdoul render log in page ', ()=>{

    const wrapper = shallow(<login/>)
    expect(wrapper).toMatchSnapshot()

})