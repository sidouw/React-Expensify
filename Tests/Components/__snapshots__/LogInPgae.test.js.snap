// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shdoul render log in page  1`] = `<login />`;
